src/main/java/com/example/GymProject/model/User.java

```diff
@@ -1,93 +1,32 @@
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "users")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
 
+    @Column(name = "first_name", nullable = false)
     private String firstName;
-    private String lastName;
-    private String username;
-    private String password;
-    private Boolean isActive;
-    public User() {
-    }
-    public User(String firstName, String lastName, String username, String password, Boolean isActive) {
-        this.firstName = firstName;
-        this.lastName = lastName;
-        this.username = username;
-        this.password = password;
-        this.isActive = isActive;
-    }
-    public User(String firstName, String lastName, Boolean isActive) {
-        this.firstName = firstName;
-        this.lastName = lastName;
-        this.password = Utils.generatePassword();
-        this.isActive = isActive;
-    }
-
-    public String getFirstName() {
-        return firstName;
-    }
-
-    public void setFirstName(String firstName) {
-        this.firstName = firstName;
-    }
-
-    public String getLastName() {
-        return lastName;
-    }
-
-    public void setLastName(String lastName) {
-        this.lastName = lastName;
-    }
 
-    public String getUsername() {
-        return username;
-    }
-
-    public void setUsername(String username) {
-        this.username = username;
-    }
-
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
-
-    public Boolean getActive() {
-        return isActive;
-    }
-
-    public void setActive(Boolean active) {
-        isActive = active;
-    }
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
 
-    @Override
-    public String toString() {
-        final StringBuilder sb = new StringBuilder("User{");
-        sb.append("firstName='").append(firstName).append('\'');
-        sb.append(", lastName='").append(lastName).append('\'');
-        sb.append(", username='").append(username).append('\'');
-        sb.append(", password='").append(password).append('\'');
-        sb.append(", isActive=").append(isActive);
-        sb.append('}');
-        return sb.toString();
-    }
+    @Column(name = "username", nullable = false, unique = true)
+    private String username;
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        User user = (User) o;
-        return Objects.equals(firstName, user.firstName) && Objects.equals(lastName, user.lastName) && Objects.equals(username, user.username) && Objects.equals(password, user.password) && Objects.equals(isActive, user.isActive);
-    }
+    @Column(name = "password", nullable = false)
+    private String password;
```

Comment:
What about storing passwords directly in db? Is it a recommended approach? I think we need to discuss this.

Reply:
I agree. In the next task, I will add a hashing algorithm and store the hashed password in the database.